contracts/package.json

```diff
@@ -5,12 +5,14 @@
   "main": "index.js",
   "scripts": {
     "compile": "hardhat starknet-compile",
+    "compile:l1": "npx hardhat compile",
```

Comment:
Why npx if we have it as a dependency?

```suggestion
    "compile:l1": "hardhat compile",
```

Reply:
It seems to be the way hardhat scripts are executed everywhere